connector-api/src/main/java/it/pagopa/selfcare/external_interceptor/connector/model/institution/PspData.java

```diff
@@ -10,6 +10,6 @@ public class PspData {
     private String legalRegisterNumber;
     private String abiCode;
     private Boolean vatNumberGroup;
-    private DpoData dpoData;
+    private DpoData dpoData;//fixme: needs to be removed
```

Comment:
you might add a task to remember that it will be removed and add here reference to task